CPPLINT.cfg

```diff
@@ -13,3 +13,4 @@ filter=-build/header_guard
 filter=-build/c++tr1
 filter=-runtime/references
 filter=-runtime/int
+filter=-build/endif_comment
```

Comment:
Why do  we need to disable this rule?

Reply:
cpplint rule : Uncommented text after #endif is non-standard. In the case of ifdef MSVC that closes 2 lines below i find this a bit extreme... I can put it back if needed

Reply:
Always better to keep coding style guards